src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingNoAuthnContextResponseAcceptanceTest.java

```diff
@@ -48,6 +51,7 @@ public void setUp() {
         complianceTool.initialiseWithDefaultsForV2();
     }
 
+    @Ignore
```

Comment:
We haven't implemented the non-happy path yet, so this test needs to be ignored; it's not a trivial fix.